### PR TITLE
Added Decimal Conversion Button and it's functionality

### PR DESCRIPTION
**Pull request by pranav-jadhav**
Issue: https://github.com/Alitindrawan24/Binary-Calculator/issues/36

Added Binary to Decimal Conversion Button and also checked for invalid conversions.

image

image

### CODE REVIEW

**Comment by Alitindrawan24** [comment]
The PR got a conflict after I merged some PR, can you solve the conflict?

File: js/binaryCalculator.js [hunk 1]
```diff
@@ -173,6 +173,17 @@ document.addEventListener('keydown', event => {
...
+  console.log("decimal" + res.value);
+  let binaryString = res.value;
+  if (/^[01]+$/.test(binaryString)) {
+    res.value = parseInt(binaryString, 2);
+    console.log("decimal" + res.value);
+    console.log("decimal" + parseInt(binaryString, 2));
+  } else {
+    res.value = "Invalid binary input";
+  }
+}
```
Comment (Alitindrawan24) [inline js/binaryCalculator.js:222]: I think the update will break the calculator if the decimal value is displayed in the calculator. Can you make the display on popup modal or something else that not break the calculator functionality ?

And also you can remove the console.log line code

Comment (pranav-jadhav) [reply js/binaryCalculator.js:222]: I will do that

**Comment by debghs** [comment]
hey @pranav-jadhav could you please wait till my PR(#53) is merged ??
I have fixed some major issues in the code in my latest PR and it is essential that the changes are made in the repo asap, as a lot of the logic is currently broken and incorrect.

**Comment by Alitindrawan24** [comment]
A conflict happen in this PR, can you fix it ? @pranav-jadhav

**Comment by pranav-jadhav** [comment]
@Alitindrawan24 
Can you please help me with this?
Please explain me what needs to be done
This is my first actual PR

**Comment by pranav-jadhav** [comment]
@Alitindrawan24 

Could you Please merge #55 and #56 first so that I can add buttons there? That would again cause a conflict.

**Comment by pranav-jadhav** [comment]
@Alitindrawan24 

Could you Please merge #55 and #56 first so that I can add buttons there? That would again cause a conflict.

**Pull request closed without merging**